backend/tracim_backend/views/core_api/schemas.py

```diff
@@ -1548,6 +1548,34 @@ class Meta:
         description = "Full workspace information"
 
 
+class EmailNotificationTypeSchema(marshmallow.Schema):
+    email_notification_type = StrippedString(
+        example=EmailNotificationType.SUMMARY.name,
+        description="Type of email notification for a specific space",
```

Comment:
what are the valid values? This should be explained in the description which is used for API documentation

Reply:
this function didn't come from this PR, the order of the function in the file just got changed

Reply:
Yes I saw this; this is a good opportunity to improve it